cpp/velox/compute/VeloxPlanConverter.cc

```diff
@@ -254,6 +254,7 @@ std::shared_ptr<gluten::ArrowArrayResultIterator> VeloxPlanConverter::GetResultI
     arrowInputIters_ = std::move(inputs);
   }
   const std::shared_ptr<const core::PlanNode> planNode = getVeloxPlanNode(plan_);
+  cacheOutputSchema(planNode);
```

Comment:
I'm not sure if we could add extra code process just for benchmark?

Reply:
Makes sense.